rpc/client.go

```diff
@@ -96,15 +95,15 @@ func (rc *RpcClient) CreateVirtual(intermediaries []types.Address, counterparty
 		rand.Uint64(),
 		common.Address{})
 
-	return waitForRequest[virtualfund.ObjectiveRequest, virtualfund.ObjectiveResponse](rc, serde.VirtualFundRequestMethod, objReq)
+	return waitForRequest[virtualfund.ObjectiveRequest, virtualfund.ObjectiveResponse](rc, serde.CreatePaymentChannelRequestMethod, objReq)
 }
 
-// CloseVirtual closes a virtual channel
-func (rc *RpcClient) CloseVirtual(id types.Destination) protocols.ObjectiveId {
+// ClosePaymentChannel closes a virtual channel
```

Comment:
```suggestion
// ClosePaymentChannel attempts to close the payment channel with supplied id
```

When we get on to working on our next grant, we should be able to drop the "attempts to" disclaimer. For now, closing a channel requires cooperative counterparties.